docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+= Static Site Generators
+
+Asciidoctor fundamentally converts one source document into one output document.
+To convert more than one document, you need some amount of infrastructure.
+Simple situations may find it convenient to use built-in Asciidoctor capabilities or scripts, but are unlikely to be appropriate for any but the smallest sites.
+Static site generators organize the conversion of multiple source documents into a functioning static web site.
+
+== What does a Static Site Generator (ssg) do?
```

Comment:
```suggestion
== What is a static site generator?
```

Reply:
I think the question of what a SSG is is too philosophical for me to answer, and all I could say was "it's something that does X", whereas if I ask what a SSG does, I can just start off explaining X.